knyst/examples/basic/adjust_frequency.rs

```diff
@@ -57,3 +48,31 @@ fn main() -> Result<()> {
     }
     Ok(())
 }
+
+/// Initializes the audio backend and starts a `KnystSphere` for audio processing.
+/// Start with an automatic helper thread for scheduling changes and managing resources.
+/// If you want to manage the `Controller` yourself, use `start_return_controller`.
+///
+/// The backend is returned here because it would otherwise be dropped at the end of setup()
+///
+/// # Returns
+///
+/// A tuple containing:
+/// - `impl AudioBackend`: The initialized audio backend, either CPAL or JACK.
+/// - `Option<SphereId>`: The optional unique identifier for the created `KnystSphere`.
```

Comment:
Comment does not describe the code anymore. I don't think the `# Returns` section is necessary (goes for all the examples). Especially without the tuple, the return value is self documenting enough.